docs/host_for_multiuser/setup.md

```diff
@@ -64,88 +66,95 @@ Follow the steps below to setup `multiuser` mode.
 8. Click "Generate new private key" in "Firebase Admin SDK" section.
 9. Put the downloaded file to `studio/config/auth/firebase_private.json`.
 
-### Setup mysql(or mariadb)
-1. Install mysql(or mariadb) server.
-2. Connect mysql server.
```

Comment:
Is Connect mysql server separate from Setup Database part? If so, maybe some more details on how.

If not, consider changing to #### Setup Database, so that Setup Database comes under Connect mysql server.